hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/Invocation.java

```diff
@@ -445,6 +445,11 @@ void notifyBackupComplete() {
 
         // We are the lucky ones since we just managed to complete the last backup for this invocation and since the
         // pendingResponse is set, we can set it on the future.
+        complete(pendingResponse);
+    }
+
+    private void complete(Object pendingResponse) {
```

Comment:
Since you pass different type of objects into the method (response, value, CALL_TIMEOUT, HEARTBEAT_TIMEOUT) I would use the same parameter name as the `AbstractInvocationFuture::complete`, which would be `value`.